Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-HAFNIUM_-_Exchange_0-day_exploits_6.yml

```diff
@@ -0,0 +1,2370 @@
+      username: {}
+    reputationcalc: 2
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 970,
+          "y": 350
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "35":
+    id: "35"
+    taskid: bd48cd7f-606f-4cec-8670-ca83f4a42e37
+    type: condition
+    task:
+      id: bd48cd7f-606f-4cec-8670-ca83f4a42e37
+      version: -1
+      name: Whether to block indicators automatically?
+      description: Checks whether IOCs associated with the incident can be blocked automatically.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "36"
+      "yes":
+      - "27"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.BlockIndicatorsAutomatically
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    view: |-
+      {
+        "position": {
+          "x": 470,
+          "y": 2330
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "36":
+    id: "36"
+    taskid: b82abb8b-34de-4af8-8eea-53c175f79552
+    type: regular
+    task:
+      id: b82abb8b-34de-4af8-8eea-53c175f79552
+      version: -1
+      name: Manually block indicators
+      description: Manually block the IOCs in the relevant systems.
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "37"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 250,
+          "y": 2510
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "37":
+    id: "37"
+    taskid: 92bd85c0-be39-4dee-8a8a-3f26f2667afd
+    type: title
+    task:
+      id: 92bd85c0-be39-4dee-8a8a-3f26f2667afd
+      version: -1
+      name: Check patch level
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "38"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 470,
+          "y": 2690
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "38":
+    id: "38"
+    taskid: ad01a248-97ee-4100-86b9-0b260f0e1633
+    type: condition
+    task:
+      id: ad01a248-97ee-4100-86b9-0b260f0e1633
+      version: -1
+      name: Check patch levels of Exchange Server
+      description: 'Is the Exchange patched with the March 2021 security updates? '
```

Comment:
Let's add link to microsoft blog here as reference for the check

Reply:
Is the Exchange patched with the March 6th, 2021 security updates?

Let's make this task name